packages/compiler/server/serverlib.ts

```diff
@@ -291,17 +293,17 @@ export function createServer(host: ServerHost): Server {
         {
           name: "<root>",
           uri: params.rootUri,
-          path: ensureTrailingDirectorySeparator(
-            resolvePath(compilerHost.fileURLToPath(params.rootUri))
-          ),
+          path: ensureTrailingDirectorySeparator(await fileURLToRealPath(params.rootUri)),
         },
       ];
     } else if (params.rootPath) {
       workspaceFolders = [
         {
           name: "<root>",
           uri: compilerHost.pathToFileURL(params.rootPath),
-          path: ensureTrailingDirectorySeparator(resolvePath(params.rootPath)),
+          path: ensureTrailingDirectorySeparator(
+            await compilerHost.realpath(resolvePath(params.rootPath))
```

Comment:
```suggestion
            await resolveRealPath(params.rootPath)
```

I think here otherwise you might endup with windows `\`